### ISSUE TITLE
Need access to the model of the Generator

### ISSUE THREAD

**Issue by YCHM0304** [arka57/DCGAN#1]
Sorry, I don't have the access to the model of the generator. Could you share it? Thank you!

**Comment by arka57** [comment]
ok Checking

**Comment by arka57** [comment]
Hi
Are you talking about the weights ? There is a mistake in the links .
Generator and Discriminator weights linked are swapped

**Comment by YCHM0304** [comment]
Yes, I mean the weights. There is no problem with the first link. However, after I clicked the second link, Google Drive asked me to request for the access.

**Comment by arka57** [comment]
Yes it is asking. I don't know why. Please can you request then I can give
access

**Comment by YCHM0304** [comment]
Requested! Thank you so much!
